web/js/validation.js

```diff
@@ -0,0 +1,4 @@
+(function () {
+    var regex = "[a-zA-Z0-9_]+";
+    $("#user_username").attr("pattern", regex);
```

Comment:
@Badlapje heb dit toegevoegd aan de formbuilder voor de registratie pagina, is dat ok?